packages/cf-deploy-config-writer/src/cf-writer/app-config.ts

```diff
@@ -238,17 +247,26 @@ async function generateDeployConfig(cfAppConfig: CFAppConfig, fs: Editor): Promi
     if (config.isMtaRoot) {
         await updateRootPackage({ mtaId: config.mtaId ?? config.appId, rootPath: config.rootPath }, fs);
     }
+    if (!skipInstall) {
+        // When installing, we need to ensure that the package.json is written to disk before running npm install
+        await new Promise((resolve) => fs.commit(resolve));
```

Comment:
Not a good idea to call `commit` as its managed by the Yeoman environment lifecycle.

Reply:
This entire flow is now removed.